Packs/Expanse/Playbooks/Expanse_Behavior_Severity_Update.yml

```diff
@@ -0,0 +1,1636 @@
+id: Expanse Behavior Severity Update
+version: -1
+name: Expanse Behavior Severity Update
+description: 'This playbook updates the severity of an Expanse Behavior incident
+  based on the presence of other active Exposures for the IP address. '
+starttaskid: "0"
+fromversion: 5.0.0
+tasks:
+  "0":
+    id: "0"
+    taskid: 7439bbfa-861b-4911-8911-6400f906a53a
+    type: start
+    task:
+      id: 7439bbfa-861b-4911-8911-6400f906a53a
+      version: -1
+      name: ""
+      description: start
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": -430
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "1":
+    id: "1"
+    taskid: f5efdc23-8d06-4c13-8ed9-e6bd27ca1904
+    type: regular
+    task:
+      id: f5efdc23-8d06-4c13-8ed9-e6bd27ca1904
+      version: -1
+      name: Expanse IP Enrichment
+      description: Submits an IP address to check.
+      script: Expanse|||ip
+      type: regular
+      iscommand: true
+      brand: Expanse
+    nexttasks:
+      '#none#':
+      - "7"
+    scriptarguments:
+      ip:
+        complex:
+          root: ${Expanse
```

Comment:
@andrew-expanse you can remove the `${` from this task since you are using 'get'